### PR TITLE
MGDOBR-1212 - Replace offline-token authentication with credentials

### PR DESCRIPTION
**Pull request by jiripetrlik**
[MGDOBR-1212](https://issues.redhat.com/browse/MGDOBR-1212)

Replace ConnectorsOidcClient offline-token authentication with credentials authentication

### CODE REVIEW

File: manager-parent/manager-v1/src/main/resources/application.properties [hunk 1]
```diff
@@ -16,6 +16,7 @@ managed-connectors.auth.server-url=${MANAGED_CONNECTORS_AUTH_SERVER_URL:fakeServ
 managed-connectors.auth.client-id=${MANAGED_CONNECTORS_AUTH_CLIENT_ID:fakeClientId}
 managed-connectors.auth.token-path=${MANAGED_CONNECTORS_AUTH_TOKEN_PATH:fakeTokenPath}
 managed-connectors.auth.offline-token=${MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN:fakeOfflineToken}
```
Comment (sutaakar) [inline manager-parent/manager-v1/src/main/resources/application.properties:18]: AFAIK this can be removed.

**Comment by sutaakar** [comment]
@jiripetrlik Please adjust also `localconfig-example`

File: dev/bin/configure.sh [hunk 2]
```diff
@@ -97,7 +97,6 @@ function configure_kafka {
 }
 
 function configure_managed_connectors {
-  check_required_variable "MANAGED_CONNECTORS_AUTH_OFFLINE_TOKEN"
```
Comment (sutaakar) [inline dev/bin/configure.sh:100]: can you check that `MANAGED_CONNECTORS_AUTH_CLIENT_ID` and `MANAGED_CONNECTORS_AUTH_CREDENTIALS_SECRET` are set?

**Review by r00ta — APPROVED** [review]
If you could send an email to our mailing list when you merge this it would be great

**Review by sutaakar — APPROVED** [review]

**Pull request merged**